cypress/support/step-definitions/carousel-steps.js

```diff
@@ -23,9 +23,17 @@ function clickCarouselButton(direction) {
   }
 }
 
-Then('slide title is {string}', (title) => {
+Then('classic slide title is {string}', (title) => {
   cy.wait(1500, { log: DEBUG_FLAG }); // required because of component refresh
-  slide().should('have.text', title);
+  classicSlide().should('have.text', title);
+});
+
+Then('{word} slide title is {string}', (index, title) => {
```

Comment:
1. This `Then` description will be treated by cucumber as the same `Then` step above. (Line 26)
2. If you want to pass `index` like a number to make for loop work correctly you should use `{int}` instead of `{word}`

Reply:
Amended to use `{int}` in the step for a corresponding integer in the feature file 👍